insitu/scripts/template/CMakeLists.txt

```diff
@@ -0,0 +1,78 @@
+cmake_minimum_required(VERSION 3.0.2)
+project({0})
+
+# specify the C++ standard
+set(CMAKE_CXX_STANDARD 14)
+set(CMAKE_CXX_STANDARD_REQUIRED True)
+
+## Find catkin macros and libraries
+## if COMPONENTS list like find_package(catkin REQUIRED COMPONENTS xyz)
+## is used, also find other catkin packages
+find_package(catkin REQUIRED COMPONENTS
+    qt_build 
+    nodelet
+    pluginlib
+    roscpp
+    insitu
+)
+find_package(OpenCV 4.2 REQUIRED)
+find_package(jsoncpp REQUIRED)
+
+## System dependencies are found with CMake's conventions
+find_package(Qt5 COMPONENTS Core Gui Widgets)
+
+catkin_package(
+#  INCLUDE_DIRS include
+#  LIBRARIES {0}
+#  CATKIN_DEPENDS nodelet pluginlib roscpp
+#  DEPENDS system_lib
+)
```

Comment:
The intent here is for users to uncomment as needed right? I like that, but maybe the `INCLUDE_DIRS` line should be uncommented automatically in the template?

Reply:
Good catch, pushed a new commit to fix. Needs testing on Noetic before I follow through with merge.